### PR TITLE
fix: swipedown gesture was registering when interacting with rest of the trade checklist

### PR DESCRIPTION
**Pull request by kaladivo**

### CODE REVIEW

**Comment by kaladivo** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #769** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/769?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/vexl-it/vexl/769?utm_source=stack-comment).

File: apps/mobile/src/components/PageWithNavigationHeader/index.tsx [hunk 1]
```diff
@@ -24,36 +17,12 @@ interface Props {
...
   const {bottom} = useSafeAreaInsets()
 
-  const GoBackOnSwipeDownOrFragment = fullScreen
-    ? React.Fragment
-    : GoBackOnSwipeDown
-
   return (
     <KeyboardAvoidingView>
-      <GoBackOnSwipeDownOrFragment>
+      <>
```
Comment (SamTremko) [inline apps/mobile/src/components/PageWithNavigationHeader/index.tsx]: maybe delete useless element wrapper

**Review by SamTremko — APPROVED** [review]

File: apps/mobile/src/components/PageWithNavigationHeader/index.tsx (hunk 1 above)
Comment (kaladivo) [reply apps/mobile/src/components/PageWithNavigationHeader/index.tsx:25]: Removed the element alltogether

**Comment by kaladivo** [comment]
### Merge activity

* **Jan 30, 12:19 PM**: @kaladivo merged this pull request with [Graphite](https://app.graphite.dev/github/pr/vexl-it/vexl/769).

**Pull request merged**